2020-02-Spring/0_arrays_matrix/README.md

```diff
@@ -141,21 +153,122 @@ Go to [Top](#top)
 <!-- Don't remove -->
 <a name="s3"/>
 
-### 3. SOLUTION 3 TODO :bug:
-
-Source: TODO :bug:
-
-#### Naive/Simple Solution 
-
-TODO :bug:
-
-#### Optimal Solution
-
-TODO :bug:
+### 3. Word Search Solution 
+
+#### Solution
+
```

Comment:
_Strongly_ recommend a short (1 or 2 sentences) explanation stating that you're using DFS to search the grid.